index.js

```diff
@@ -18,6 +22,9 @@ module.exports = function (toLoad, mocks) {
 
   return mocked
 }
+var force = module.exports.force =  function(toLoad, mocks){
```

Comment:
`force` isn't very descriptive– I'd rather call this something like `andClearCache`.